gql/resolvers/patchNotes/types.js

```diff
@@ -0,0 +1,71 @@
+import { gql } from 'apollo-server'
+
+const patchNoteTypes = gql`
+  scalar Date
+  enum ErrorTypes {
+    duplicate
+    failure
+    invalid
+    notFound
+    unauthorized
+  }
+
+  type Errors {
+    type: ErrorTypes
+    message: String
+  }
+
+  type PatchNote {
+    _id: ID!
+    title: String!
+    description: String!
+    datePublished: Date
+    display: Boolean
+    targetDate: Date
+    targetVersion: Float
+    graphic: String
```

Comment:
we snappin pics ? sami's  wire frame gonna be on point

Reply:
Possibly. Built it in incase we ever wanted to add snapshots etc